### PR TITLE
Pandance prototype project

### PR DESCRIPTION
**Pull request by yurynix**
A prototype project.
   
#### Testing instructions
  
1. Run `git checkout add/pandance` and start your server, or open a [live branch](https://calypso.live/?branch=add/pandance)
2. Open[`Pandance`](http://calypso.localhost:3000/pandance)

### CODE REVIEW

**Comment by gziolo** [comment]
Can you add Readme file inside `client/pandance ` folder explaining that it's only test? :)

File: client/pandance/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ * External dependencies
+ */
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import { removeSidebar, index, businessInfo, contentPreview, customizeBlock, createAccount } from './controller';
+
+export default function() {
```
Comment (gziolo) [inline client/pandance/index.js]: Can you hide it behind the feature flag?

**Comment by gziolo** [comment]
Actually it might be moved to `client/extensions` folder, it would allow to move Redux state changes to local folder and hide everything behind the feature flag using `package.json` file.

**Comment by fabianapsimoes** [comment]
@mikeshelton1503 seems this is pending a review from you, according to our board. Do you have an idea when you'll be able to get to it?

**Comment by mikeshelton1503** [comment]
Oh sorry, didn't realize this was an actual real branch now. It's just a prototype. The changes @yurynix made look good. Nothing further here. Just need to finish the post and then we can share.

**Comment by yurynix** [comment]
It was a real branch before as well, just not a PR, which is easier to track IMO.
We're still not planning to merge that =)

**Comment by apeatling** [comment]
I don't see anything being merged here, or any real activity. Please reopen if this changes.

**Pull request closed without merging**